src/main/java/com/codurance/allaboard/survey/services/SaveSurvey.java

```diff
@@ -0,0 +1,21 @@
+package com.codurance.allaboard.survey.services;
+
+import com.codurance.allaboard.survey.infrastructure.repositories.Surveys;
+import com.codurance.allaboard.survey.model.Survey;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SaveSurvey {
+
+  private final Surveys surveys;
+
+  @Autowired
+  public SaveSurvey(Surveys surveys) {
+    this.surveys = surveys;
+  }
+
+  public void saveSurvey(Survey survey) {
```

Comment:
Maybe this method could be renamed to something different to avoid having `saveSurvey.saveSurvey(survey)` calls.